docs/specs/RIPR-SPEC-0157-changed-binding-predicate-relation.md

```
@@ -0,0 +1,124 @@
+# RIPR-SPEC-0157: Changed-binding → predicate-operand relation
+
+Status: proposed
+
+Issue: #3294 (parent #3215; follows #3271)
+
+## Problem
+
+#3271 made the `find`/`rfind`/`len_utf8` through `map_or` boundary
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)